README.md

```diff
@@ -85,26 +85,33 @@ OPTIONS
 Vegastrike takes a single command parameter indicating which mission it should load
 
 ```bash
-bin/vegastrike mission/explore_universe.mission
+bin/vegastrike-engine mission/explore_universe.mission
 ```
 
 is an example of a valid mission call
 
 the -l flag (must be flushed with the system) will force a player to begin in a star system.
 
 ```bash
-bin/vegastrike -lvega_sector/vega mission/bomber.mission
+bin/vegastrike-engine -lvega_sector/vega mission/bomber.mission
```

Comment:
needs the data_dir option specified in this mode

Reply:
Another good catch. Thanks

Reply:
Fixed